app/dao/fact_billing_dao.py

```diff
@@ -457,6 +458,10 @@ def get_rate(
     if notification_type == LETTER_TYPE:
         if letter_page_count == 0:
             return 0
+
+        if post_class in INTERNATIONAL_POSTAGE_TYPES:
```

Comment:
Optional, if we are to keep this approach then I would suggest an inline comment here saying
'we only have one billing rate for international letters so all postage types in INTERNATIONAL_POSTAGE_TYPES get mapped to it`

Reply:
This commit has been removed, since we're now adding a separate row for `europe` and `rest-of-world` letters to the `letter_rates` table.